src/storage/txn/process.rs

```diff
@@ -209,7 +61,7 @@ fn extract_lock_from_result<T>(res: &StorageResult<T>) -> Lock {
     }
 }
 
-pub(super) struct WriteResult {
+pub struct WriteResult {
```

Comment:
Same with WriteResult.

Reply:
> Same with WriteResult.

`WriteResult` is also used in scheduler, we cannot remove the `super` here for now.